docs/public/specifications/test-zones/Zone-TP/zone11.md

```diff
@@ -0,0 +1,325 @@
+| NO-SPF-TXT             | Z11_NO_SPF_FOUND                                                | 2)                     |
+| NO-SPF-ROOT-ZONE       | Z11_NO_SPF_NON_MAIL_DOMAIN                                      | 2)                     |
+| NO-SPF-TLD-ZONE        | Z11_NO_SPF_NON_MAIL_DOMAIN                                      | 2)                     |
+| NO-SPF-ARPA-ZONE       | Z11_NO_SPF_NON_MAIL_DOMAIN                                      | 2)                     |
+| NULL-SPF-ROOT-ZONE     | Z11_NULL_SPF_NON_MAIL_DOMAIN                                    | 2)                     |
+| NULL-SPF-TLD-ZONE      | Z11_NULL_SPF_NON_MAIL_DOMAIN                                    | 2)                     |
+| NULL-SPF-ARPA-ZONE     | Z11_NULL_SPF_NON_MAIL_DOMAIN                                    | 2)                     |
+| NON-NULL-SPF-ROOT-ZONE | Z11_NON_NULL_SPF_NON_MAIL_DOMAIN                                | 2)                     |
+| NON-NULL-SPF-TLD-ZONE  | Z11_NON_NULL_SPF_NON_MAIL_DOMAIN                                | 2)                     |
+| NON-NULL-SPF-ARPA-ZONE | Z11_NON_NULL_SPF_NON_MAIL_DOMAIN                                | 2)                     |
+| INVALID-SYNTAX-1       | Z11_SPF_SYNTAX_ERROR                                            | 2)                     |
+| INVALID-SYNTAX-2       | Z11_SPF_SYNTAX_ERROR                                            | 2)                     |
+| INVALID-SYNTAX-3       | Z11_SPF_SYNTAX_ERROR                                            | 2)                     |
+| NON-AUTH-TXT           | Z11_UNABLE_TO_CHECK_FOR_SPF                                     | 2)                     |
+| SERVFAIL               | Z11_UNABLE_TO_CHECK_FOR_SPF                                     | 2)                     |
+| INCONSISTENT-SPF       | Z11_INCONSISTENT_SPF_POLICIES, Z11_DIFFERENT_SPF_POLICIES_FOUND | 2)                     |
+| SPF-MISSING-ON-ONE     | Z11_INCONSISTENT_SPF_POLICIES, Z11_DIFFERENT_SPF_POLICIES_FOUND | 2)                     |
+| ALL-DIFFERENT-SPF      | Z11_INCONSISTENT_SPF_POLICIES, Z11_DIFFERENT_SPF_POLICIES_FOUND | 2)                     |
+| MULTIPLE-SPF-RECORDS   | Z11_SPF_MULTIPLE_RECORDS                                        | 2)                     |
+
+
+* (1) All tags except for those specified as "Forbidden message tags" (no instances for these test scenarios)
+* (2) All tags except for those specified as "Mandatory message tags"
+
+## Test scenarios and setup of test zones
+
+### Default zone configuration
+
+Unless otherwise specified in the specific scenario specification, the test zone
+for the scenario will follow the default setup as stated below. The `child zone`
+is the zone to be tested for the scenario.
+
+* The child zone is `SCENARIO.zone11.xa`.
+  * There is a zone file for the child zone.
+  * It is served by three name servers named ns1, ns2 and ns3.child.zone11.xa.

```

Comment:
For the default scenarios I include one or two NS to make configuration smaller. What do you think will be gained by three?

Reply:
There are two scenarios where I really need three: SPF-MISSING-ON-ONE and ALL-DIFFERENT-SPF. For all other scenarios, two are enough.